libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -282,32 +282,26 @@ class MapboxTripNotification constructor(
     }
 
     private fun getManeuverResource(step: LegStep): Int {
-        val maneuver = step.maneuver()
-        val maneuverType = maneuver.type()
-        maneuver.let { stepManeuver ->
-            val maneuverModifier = stepManeuver.modifier()
-            if (!TextUtils.isEmpty(maneuverModifier)) {
-                val drivingSide = step.drivingSide()
-                val isLeftSideDriving = isLeftDrivingSideAndRoundaboutOrRotaryOrUturn(
-                    maneuverType,
-                    maneuverModifier,
-                    drivingSide
-                )
-                return when (isLeftSideDriving) {
-                    true -> {
-                        ManeuverResource.obtainManeuverResource(
-                            maneuverType + maneuverModifier + drivingSide
-                        )
-                    }
-                    else -> {
-                        ManeuverResource.obtainManeuverResource(
-                            maneuverType + maneuverModifier
-                        )
-                    }
-                }
+        val stepManeuver = step.maneuver()
+        val maneuverType = stepManeuver.type()
+        val maneuverModifier = stepManeuver.modifier()
+        val isModifierNotEmpty = TextUtils.isEmpty(maneuverModifier).not()
```

Comment:
kotlin has extensions like `String.isEmpty` and `String.isNotEmpty`
```suggestion
        if (maneuverModifier.isNotEmpty) {
```

Reply:
that's for non nullable types

Reply:
`Sting.isNullOrEmpty()`

Reply:
`if (maneuverModifier?.isNotEmpty == true)`

Should work as well.